src/main/java/org/ozwillo/dcimporter/service/SynchronizerService.java

```diff
@@ -9,37 +12,44 @@
 import org.springframework.stereotype.Component;
 
 @Component
-public class SynchronizerService implements CommandLineRunner{
+public class SynchronizerService implements CommandLineRunner {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(SynchronizerService.class);
-	
-    private final SystemUserService systemUserService;
-    private final DatacoreClient datacoreClient;
-    private final PublikService publikService;
-
-    @Value("${publik.datacore.project}")
-    private String datacoreProject;
-    @Value("${publik.datacore.modelEM}")
-    private String datacoreModel;
-
-    @Autowired
-	public SynchronizerService(SystemUserService systemUserService, DatacoreClient datacoreClient, PublikService publikService) {
+
+	private final SystemUserService systemUserService;
+	private final DatacoreClient datacoreClient;
+	private final PublikService publikService;
+
+	@Value("${publik.datacore.project}")
+	private String datacoreProject;
+	@Value("${publik.datacore.modelEM}")
+	private String datacoreModelEM;
+	@Value("${publik.datacore.modelSVE}")
+	private String datacoreModelSVE;
+
+	@Autowired
+	public SynchronizerService(SystemUserService systemUserService, DatacoreClient datacoreClient,
+	        PublikService publikService) {
 		this.systemUserService = systemUserService;
 		this.datacoreClient = datacoreClient;
-        this.publikService = publikService;
-    }
+		this.publikService = publikService;
+	}
 
 	public void run(String... args) {
-    	systemUserService.runAs(() -> {
-    		if (datacoreClient.findResources(datacoreProject, datacoreModel).isEmpty())
-                try {
-                    publikService.syncPublikForms();
-                    LOGGER.debug("Requests successfully synchronized");
-                } catch (Exception e) {
-                    LOGGER.error("Unable to synchronize past requests", e);
-                }
-            else
-                LOGGER.debug("Requests are already synchronized");
-    	});
-    }
+		systemUserService.runAs(() -> {
+			String formTypes[] = new String[] { datacoreModelEM, datacoreModelSVE };
```

Comment:
A more concise way of doing : Arrays.asList(datacoreModelEM, datacoreModelSVE).forEach(...)